example/lib/line_chart/line_chart_page3.dart

```diff
@@ -20,6 +21,17 @@ class LineChartPage3 extends StatelessWidget {
               height: 52,
             ),
             LineChartSample6(),
+            const SizedBox(
+              height: 52,
+            ),
+            const Text(
+              'LineChart (positive and negative values)',
+              style: TextStyle(fontSize: 32, fontWeight: FontWeight.bold, color: Colors.black),
+            ),
+            const SizedBox(
+              height: 52,
+            ),
+            LineChartSample9(),
```

Comment:
Wrap them with a ListView (instead of current Column)

Reply:
Or I can do that myself after we merge.
Because I want to show the sample in our samples section.